test/index.js

```diff
@@ -313,4 +314,434 @@ describe('Token', function () {
     done();
   });
 
+  describe('when audience is specified for validation', function(){
```

Comment:
please, update to `'when a single audience is...'` to clearly diff from array case. Since this one shows up first, I was looking for the array tests in this describe and they are in another one (which is good)